src/Teams/Services/ManageTeamsMembersService.cs

```diff
@@ -1,8 +1,29 @@
+using Teams.Repository;
+using Teams.Models;
+using Teams.Data;
+using Teams.Security;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
 namespace Teams.Services
 {
     public class ManageTeamsMembersService : IManageTeamsMembersService
     {
+        private readonly IRepository<TeamMember, int> _teamMemberRepository;
 
+        public ManageTeamsMembersService(IRepository<TeamMember, int> teamMemberRepository)
+        {
+            _teamMemberRepository = teamMemberRepository;
+        }
 
+        public async Task<TeamMember> GetMemberAsync(int team_id, string member_id)
+        {
+             return await _teamMemberRepository.GetAll()
```

Comment:
lets see, you created new service which by logic make sense, but you can use here team repository and include team members, get a team with team members and compare result with owner and list of teammembers in team in one go